frb_example/benches/dart/lib/async_benchmark.dart

```diff
@@ -0,0 +1,158 @@
+export 'async_benchmark.io.dart' if (dart.library.html) 'async_benchmark.web.dart' show AsyncBenchmark;
+
+import 'package:flutter_rust_bridge_benchmark/bridge_definitions.dart';
+import 'package:flutter_rust_bridge_benchmark/constants.dart';
+import 'package:flutter_rust_bridge_benchmark/env/stub.dart' show itemsCount, outputFilename;
+
+import 'walltime.dart';
+
+extension UnitSymbol on Unit {
+  String symbol() {
+    switch (this) {
+      case Unit.Milliseconds:
+        return 'ms';
+      case Unit.Microseconds:
+        return 'μs';
+      case Unit.Nanoseconds:
+        return 'ns';
+    }
+  }
+}
+
+class Routine {
+  final int iterations;
+  final double elapsed;
+  const Routine(this.iterations, this.elapsed);
+}
+
+class Sample {
+  final List<Routine> routines;
+  final Unit unit;
+  final String name;
+  const Sample(this.routines, this.name, this.unit);
+  int get iterations => routines.fold(0, (previousValue, element) => previousValue + element.iterations);
+  double get elapsed => routines.fold(0, (previousValue, element) => previousValue + element.elapsed);
+  Map<String, dynamic> toJson() {
```

Comment:
uber-nit: it is a common practice to use `json_serializable` instead of manually writing it

Reply:
Utterly, but since I have only one or two in the whole benches project I thought I might as well write them manually instead of adding an additional library and build command.